app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -36,6 +39,16 @@ class CursorView(context: Context, attrs: AttributeSet) : ImageView(context, att
         resetCountdown()
     }
 
+    fun updateCursorPressedState(event: KeyEvent) {
+        val remoteKey = RemoteKey.fromKeyEvent(event)
+        if (event.action == KeyEvent.ACTION_DOWN && (remoteKey == RemoteKey.CENTER
+                        || event.keyCode == KeyEvent.KEYCODE_ENTER)) { // For keyboard and emulator use.
+            setImageResource(R.drawable.cursor_full_active)
+        } else {
```

Comment:
nit: You're assuming the argument passed in will either be ACTION_DOWN or ACTION_UP and no other key presses. However, if this code is refactored by another developer, they may not know about your assumption and the `else` case might start catching other actions!

You should use an explicit else case here, or restrict your arguments (either through types or assertion-like expressions) so that your assumptions are either removed or explicitly documented.

fwiw, if this method was called `onSelectKeyPressed`, it would be another way of documenting this assumption but I think maybe you're right that this is better called updateCursorPressedState. :)